baselines/jft/active_learning.py

```diff
@@ -206,6 +318,10 @@ def finetune(*,
              ds,
              rngs_loop,
              total_steps,
+             train_ds,
```

Comment:
Add `train_ds` to the docstring below. Also, I think it would helpful to name this `train_eval_ds` to distinguish from a normal shuffled training dataset.